internal/tiltfile/docker_compose.go

```diff
@@ -57,12 +57,14 @@ func (dcm dcResourceMap) ServiceCount() int {
 func (s *tiltfileState) dockerCompose(thread *starlark.Thread, fn *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
 	var configPaths starlark.Value
 	var projectName string
+	var profiles value.StringOrStringList
```

Comment:
in this PR, the default behavior is always "no profiles". is that right?

maybe the default behavior should be "read the profiles from the COMPOSE_PROFILES env var"

what do you think?

Reply:
I think it depends on how much tilt wants the local environment to colour what tilt does versus what is defined in a `Tiltfile`. I'm not super familiar with the project so I don't know how much or if this already happens. From my perspective though the usage of `COMPOSE_PROFILES` or the `--profile` argument can be options of docker compose but not necessarily of tilt.

For my own setup I find value in the straightforwardness of my full definition being inside the `Tiltfile`. I think this also works especially well for when I have multiple `docker-compose.yml` files I'm loading and I want to be intentional about which services I need from each.

I have to say though that the setup I'm using is relatively simplistic so if there are arguments for using the `COMPOSE_PROFILES` environment variable I probably wouldn't have run into them anyways :sweat_smile:

Reply:
Ah I forgot, to answer your question: it is correct that the default behaviour is no profiles.